src/orders/components/OrderProductAddDialog/OrderProductAddDialog.tsx

```diff
@@ -267,18 +269,18 @@ const OrderProductAddDialog: React.FC<OrderProductAddDialogProps> = props => {
           }}
         />
       </DialogContent>
-      <DialogContent className={classes.content}>
+      <DialogContent className={classes.content} id={scrollableTargetId}>
         <InfiniteScroll
-          pageStart={0}
-          loadMore={onFetchMore}
+          dataLength={productChoicesWithValidVariants?.length}
+          next={onFetchMore}
           hasMore={hasMore}
-          useWindow={false}
+          scrollThreshold="100px"
```

Comment:
We went from `10` to `100` in this particular component, is this intended?

Reply:
I've changed it as imo 10 is unreasonably small